it/src/test/java/org/corfudb/universe/scenario/NodeUpAndPartitionedIT.java

```diff
@@ -110,6 +110,8 @@ public void nodeUpAndPartitionedTest() {
                 }
                 assertThat(clusterStatusReport.getClusterStatus()).isEqualTo(ClusterStatus.STABLE);
 
+                Sleep.sleepUninterruptibly(Duration.ofSeconds(10));
```

Comment:
So, as you can see, that highly likely `getClusterStatus()` works incorrectly. Data path is not ready but getClusterStatus() returns OK.

Reply:
If getClusterStatus() is faulty, we shouldn't add this sleep, which would hide this bug right?

Reply:
So, then we just can't bring any changes until we fix clusterState. Since this bug is staying in code at least for 3-5 month, looks like we are ok with that.
I see the best way is to create an issue and fix it ASAP https://github.com/CorfuDB/CorfuDB/issues/2034